go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -522,16 +538,35 @@ func (bnc *BaseNetworkController) getExpectedSwitchName(pod *kapi.Pod) (string,
 	return switchName, nil
 }
 
+func (bnc *BaseNetworkController) ensureOvnPodAnnotation(pod *kapi.Pod, nadName string) (*util.PodAnnotation, error) {
```

Comment:
Suggesting other names, in case there is anything we like better
`fetchPodAnnotation`I 
`retrievePodAnnotation`

Using ensure just because that is what we are doing for kubevirt does not really convince me because that is not the most common case. But since I don't have better alternatives than those above. I am ok with it.